openshift/scripts/oc_provision_backup_s3_postgres_cronjob.sh

```diff
@@ -29,11 +29,11 @@ PROJ_TARGET="${PROJ_TARGET:-${PROJ_DEV}}"
 # Prepare variables for backups
 JOB_NAME="backup-postgres-${APP_NAME}-${SUFFIX}"
 IMAGE_NAMESPACE=${PROJ_TOOLS}
-CLUSTER_NAME="patroni-${APP_NAME}-${SUFFIX}"
+CLUSTER_NAME="wps-crunchydb-${SUFFIX}"
 
 OC_PROCESS="oc -n ${PROJ_TARGET} process -f ${TEMPLATE_PATH}/backup-s3-postgres-cronjob.yaml \
-    -p DATABASE_SERVICE_NAME=patroni-${APP_NAME}-${SUFFIX}-leader \
     -p DATABASE_DEPLOYMENT_NAME=wps-global \
+    -p CRUNCHYDB_USER=wps-crunchydb-${SUFFIX}-pguser-wps-crunchydb-${SUFFIX} \
```

Comment:
Does it still need to be passed in here if its specified in the yaml env below? More of a curiosity question

Reply:
Yes, this passes in the value as a parameter to the yaml file. When `oc process` is called these values are filled in.

Reply:
Oh I see what's happening, thanks. Monday brain no work so good